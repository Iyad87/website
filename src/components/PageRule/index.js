import styled from 'styled-components';
import { borderLight } from '../../styles/colours';
import { spacing } from '../../styles/spacing';

const PageRule = styled.hr`
  background-color: ${borderLight};
  border: none;
  height: 1px;
  margin: ${spacing(10)} 0;
  width: 100%;
`;

export default PageRule;
