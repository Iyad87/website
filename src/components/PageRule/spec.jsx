import React from 'react';
import { shallow } from 'enzyme';
import PageRule from './index';

test('components/PageRule snapshot', () => {
  const wrapper = shallow(<PageRule />);

  expect(wrapper).toMatchSnapshot();
});
