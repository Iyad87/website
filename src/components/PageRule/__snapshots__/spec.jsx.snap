// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components/PageRule snapshot 1`] = `
.c0 {
  margin: 100px 0;
}

<Rule
  className="c0"
/>
`;
